Mvp last tweaks (cleanup of structure)

linuxbandit opened the pull request:
In theory this is the working backend without frontend yet. I am going to test it now, only the api should be successful anyway. Will add a reviewer when I finish testing

i3anaan committed:
Readded oms-serviceregistry as git submodule

i3anaan committed:
Bumped oms-events versions

i3anaan committed:
Updated docker files

i3anaan committed:
Bumped oms-events-frontend version

i3anaan committed:
Added new services to the running test for Travis

linuxbandit committed:
beginning/continuing of a cleanup of the oms-docker

linuxbandit committed:
Merge branch 'MVP-remove-submodules' into MVP-cleanup-2

* MVP-remove-submodules: (36 commits)
  beginning/continuing of a cleanup of the oms-docker
  Hotfixed pgadmin not working
  Better docker cleanup commands
  More full path changes
  Fixed docker cleanup commands
  Full path for the base docker compose file
  Added check for existing core app key
  For some reason deleted files stayed there
  Bump core version
  Updated deployment script
  Add portal to the system
  Bumped logo version (#44)
  Restore everything into one folder, will think later
  reverted accidental /dev push, oops!
  Added logo print
  Portainer working out of the box + nice logo
  PROV-7
  cleanup of some files
  Re-intergrate registry and events (#33)
  Add survey service (disabled but tested working)
  ...

linuxbandit committed:
Cherry-picked d3780a

linuxbandit committed:
cherry picked 20021a

linuxbandit committed:
Should be all integrated

This changeset should be the end of the integration and merging between stuff.
In particular: from dev (78a36) I removed most submodules (find it at MVP-remove-submodules)
Then from there I merged into last of Derk's stabilising commits (d986f), find them at MVP-cleanup-2.
From there I cherry-picked some comments, then  checked the submodules and the files that have been overwritten or changes that haven't been ported because they were in yet some other branch that wasn't involved

This is a note to self because the process was tedious as quite manual

serge1peshcoff commented on .env.example:
```diff
@@ -1,14 +1,14 @@
-PATH_OMS_EVENTS_FRONTEND=oms-events-frontend/docker/
-PATH_ALASTAIR=alastair/docker/
+PATH_OMS_CORE_ELIXIR=oms-core-elixir/docker/
+PATH_OMS_LOGINSERVICE=oms-loginservice/docker/
```
do we need login service? since it's not used and it's in the core now

linuxbandit replied:
i included it just to be sure but I agree with you

serge1peshcoff commented on .gitmodules:
```diff
@@ -1,21 +1,18 @@
+[submodule "oms-core-elixir"]
+	path = oms-core-elixir
+	url = https://github.com/AEGEE/oms-core-elixir.git
+[submodule "oms-loginservice"]
```
and here as well

serge1peshcoff commented on .env.example:
```diff
@@ -1,14 +1,14 @@
 
 # To enable services, add them to the array, like
-ENABLED_SERVICES=oms-global:oms-core:oms-serviceregistry:oms-events-frontend:oms-events:alastair:oms-logo-print
+ENABLED_SERVICES=oms-global:oms-core:oms-serviceregistry:oms-events:oms-logo-print
```
did you mean `oms-core-elixir` instead of `oms-core`? :D

linuxbandit replied:
thanks! I will correct it

linuxbandit committed:
correct oms-core version and bump oms-events

linuxbandit committed:
Bump core-elixir version

linuxbandit committed:
Removed the last submodule and tested, the PR should be complete

linuxbandit committed:
Remove https for two reasons

1) breaks the usage of api with curl and stuff to test them
2) it's gonna be reintroduced with let's encrypt support

linuxbandit committed:
Correct the configuration for traefik

or it doesn't work. Mysteries of this thing...

linuxbandit committed:
fixing travis tests

linuxbandit committed:
fixing tabs ->spaces :p

linuxbandit committed:
fixing travis tests - gahhh

linuxbandit committed:
fixing travis tests - gahhh

linuxbandit committed:
bump oms-events

linuxbandit committed:
'Do not use exit in travis script' says the docs..

linuxbandit committed:
Add frontend
